src/_includes/components/code-copy.css

```diff
@@ -0,0 +1,35 @@
+/* Copy button */
+/* clean-css doesn’t support CSS Nesting: https://github.com/clean-css/clean-css/issues/1254 */
+/* clean-css ignore:start */
+:is(syntax-highlight, .syntax-highlight):has(wa-copy-button) {
+	position: relative;
+
+	wa-copy-button {
+		position: absolute;
+		right: -1rem;
```

Comment:
Can someone in a rtl-language (Hebrew, Arabic etc) check how it looks like in their browser?